roles/collectd/tasks/main.yml

```diff
@@ -1,12 +1,17 @@
 ---
 - name: install collectd
-  yum:
-      name: '{{ item }}'
-      state: present
-  with_items:
-      - collectd
```

Comment:
@paramite This comment has been made on the code to be removed, if you look at the PR the code actually rely on `collectd_package_name`